maps/away/away_site/abandoned_industrial/abandoned_industrial_station.dm

```diff
@@ -8,7 +8,8 @@
 	spawn_weight = 1
 	suffixes = list("away_site/abandoned_industrial/abandoned_industrial_station.dmm")
 
-	sectors = list(ALL_POSSIBLE_SECTORS)
+	sectors = list(ALL_TAU_CETI_SECTORS, ALL_COALITION_SECTORS, SECTOR_BADLANDS, SECTOR_VALLEY_HALE, SECTOR_GAKAL, SECTOR_LIGHTS_EDGE)
+	sectors_blacklist = list(SECTOR_TAU_CETI, SECTOR_BURZSIA, SECTOR_HANEUNIM)
```

Comment:
TC and Haneumin okay, but why Burzsia? It's the "ship graveyard and dumping area".

Reply:
This is a fair point, will change it.